src/renderer/core/reducer.spec.ts

```diff
@@ -1,20 +1,13 @@
 /* eslint-disable @typescript-eslint/camelcase */
 import chai from 'chai';
-import * as TD from '../model/mock/testData';
-import * as types from '../store/types';
-import * as Actions from './actions';
-import {
-  initialClientData,
-  clientDataReducer,
-  initialServerData,
-  serverDataReducer
-} from './reducer';
+// import * as TD from '../model/mock/testData';
```

Comment:
Please remove the file completely or fix the test (if it's easy).

Reply:
It seems to be passing, so I uncommented it